ssts/segmentation/example.py

```diff
@@ -206,6 +187,41 @@ def main(data_path=None, example_number=None):
         # Plot classification distributions with masks
         mm.show_distributions_together(labels, data)  # all classes plotted together
 
+    # NOTE Wes workflow
+    elif example_number == "7":
+
+        # Apply frequency removal
+        data = mm.apply_frequency_removal(data)
```

Comment:
Low frequency removal step

Reply:
This step has been improved to filter according to frequency magnitude as opposed to within some radius